spanner/spanner_samples.rb

```diff
@@ -1009,6 +1041,7 @@ def usage
   write_with_transaction_using_dml   <instance_id> <database_id> Update data using a DML statement within a read-write transaction.
   update_using_partitioned_dml       <instance_id> <database_id> Update multiple records using a partitioned DML statement.
   delete_using_partitioned_dml       <instance_id> <database_id> Delete multiple records using a partitioned DML statement.
+  update_using_batch_dml             <instance_id> <database_id> Updates sample data in the database using Batch DML.
```

Comment:
Please update the README.md with the new CLI command.

Reply:
Done.